### PR TITLE
RSWEB-9503: Page Banner

### PR DESCRIPTION
**Pull request by brantdarilek**
For Review:
http://rsweb-9503-page-banner.styleguide.dev.website.rackspace.com/derek/incubation/refresh

### CODE REVIEW

File: styleguide/derek/incubation/refresh.jade [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+
+.half-padding-full
+  !=partial("../pattern-components/headers/header-refresh", {desc: "the cloud keeps you ahead of the curve, <br/>but what if it throws you a few", size:"Medium"})
+
+.half-padding-full
+  !=partial("../pattern-components/headers/header-refresh", {desc: "3,000 cloud experts working for you 24/7? <br/>Sounds like you may need a bigger office", size:"Medium"})
+
+.half-padding-full
+  !=partial("../pattern-components/headers/header-refresh", {desc: "1 B hours managing openstack clouds? <br/>Talk about putting time on your side", size:"Medium"})
```
Comment (WidgetsBurritos) [inline styleguide/derek/incubation/refresh.jade]: Not sure if just a content problem but just wanna sure we're aware.  This text gets cut off oddly at around 820px.

image

**Comment by WidgetsBurritos** [comment]
Oh also firefox looks funky

image

**Comment by brantdarilek** [comment]
@WidgetsBurritos 
I've addressed your feedback and this is ready for review.

**Comment by WidgetsBurritos** [comment]
@brantdarilek can you rebase this so we can get rid of amber's commit from the history here and also ensure you're up-to-date with the latest master?

**Pull request merged**